staff/sys/makevm

```
@@ -262,7 +276,12 @@ def _main(args):
     print('\tOS Type: {}'.format(args.os_type))
     print('\tOS Variant: {}'.format(args.os_variant))
     print('\tDisk Space: {} GB'.format(args.storage))
-    print('\tVolume Group: {}'.format(args.vg))
+    if args.ceph:
```

we should throw an exception if we are passed both args.